### ISSUE TITLE
Add supported engine version in package.json

### ISSUE THREAD

**Issue by gepdev** [Capgemini/dcx-react-library#568]
**Is your feature request related to a problem? Please describe.**
I received multiple errors and the application wasn't working when using Node version 18.18.2. Then after I have changed the version to 16.20.2 and the setup script completed correctly.

**Describe the solution you'd like**
Would be good having the engine at least on the `package.json` so as a developer I can see what is the engine versions supported in order to have the local environment configured properly for the application to run
Example:
`"engines": {
    "node": ">=16.16.0 <17",
    "npm": ">=7.20.0 <9",
    "yarn": ">=1.22.1 <2"
  },`

**Comment by daniele-zurico** [comment]
hi @gepdev given you got the errors and the minimum configuration version can you please share your nodeJS and yarn version as a starting point? We can use that one and then keep it updated as soon as we progress.
Just out of curiousity mine is:
```bash
danielezurico@Danieles-MacBook-Pro ~ % node --version
v18.14.0
danielezurico@Danieles-MacBook-Pro ~ % yarn --version
1.22.10
danielezurico@Danieles-MacBook-Pro ~ % npm --version
9.3.1
```
and everything works as expected

**Comment by gepdev** [comment]
> hi @gepdev given you got the errors and the minimum configuration version can you please share your nodeJS and yarn version as a starting point? We can use that one and then keep it updated as soon as we progress. Just out of curiousity mine is:
> 
> ```shell
> danielezurico@Danieles-MacBook-Pro ~ % node --version
> v18.14.0
> danielezurico@Danieles-MacBook-Pro ~ % yarn --version
> 1.22.10
> danielezurico@Danieles-MacBook-Pro ~ % npm --version
> 9.3.1
> ```
> 
> and everything works as expected

I retried again I suppose I was having issues with Yarn version which in my Node (18.18.2) environment was in version 3. So I have downgraded it to v1 (1.22.11)

**Comment by daniele-zurico** [comment]
yes @gepdev is the yarn 3 that we do not support.... I'll say that for the ticket we can put the 3 numbers that I got listed

**Comment by daniele-zurico** [comment]
closed by #570
